opengemini-client-common/src/main/java/io/opengemini/client/common/ResultMapper.java

```diff
@@ -0,0 +1,40 @@
+package io.opengemini.client.common;
+
+import io.opengemini.client.api.QueryResult;
+import io.opengemini.client.api.RetentionPolicy;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ResultMapper {
+
+    public static List<String> toDatabases(QueryResult result) {
+        return result.getResults().get(0).getSeries().get(0).getValues().stream().map(
+                x -> String.valueOf(x.get(0))).collect(Collectors.toList());
+    }
+
+    public static List<RetentionPolicy> toRetentionPolicies(QueryResult result) {
+        return converseRps(result.getResults().get(0).getSeries().get(0).getValues());
+    }
+
+    private static List<RetentionPolicy> converseRps(List<List<Object>> queryRpValues) {
+        ArrayList<RetentionPolicy> retentionPolicies = new ArrayList<>();
```

Comment:
The type of variable definition can be written as List

Reply:
OK